pkgs/data/fonts/terminus-font/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, python3
 , bdftopcf, mkfontscale
+, variants ? []
```

Comment:
Can we make this an enum style input for all the variants listed here https://github.com/bachan/terminus-font-vietnamese/tree/master/alt ?

Reply:
@SuperSandro2000 Is there a good example how to use `types.enum` at this level? Enums don't seem to be a documented feature of Nix language and I don't see that enums were used anywhere in pkgs/

Reply:
see https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/themes/matcha/default.nix for an example